tests/parsenum/main.c

```diff
@@ -1,4 +1,3 @@
-#include <inttypes.h>
```

Comment:
You're going to want it when you declare a bunch more variables and parse numbers into them. :-)